addons/l10n_hr/views/l10n_hr_kpd_category_views.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="l10n_hr_kpd_category_view_tree" model="ir.ui.view">
```

Comment:
I note that we create these views but don't have any action to open them. Maybe it would be worth creating a menuitem in Accounting -> Configuration that shows in debug mode (you can achieve that by putting `groups="base.group_no_one"` on the menuitem)